source/2_platform/rex_opengl/include/rex_opengl/resources/buffer.h

```diff
@@ -16,8 +16,13 @@ namespace rex
     {
     public:
         Buffer(u32 target);
+        Buffer(const Buffer&) = delete;
+        Buffer(Buffer&&) = default;
```

Comment:
the default move ctor will copy the resource id without clearing the one of the input.
I don't think this is intended, please define a custom implementation that does this.